### ISSUE TITLE
toast notifications coming through as raw.

### ISSUE THREAD

**Issue by elliotstokes** [tjanczuk/wns#14]
Great plugin. We are currently working on getting wns notifications working for our app and are having a few problems.

When the notification is sent we receive the notification when the app is open and the toasts are coming through as raw. If the app is closed then we see no notification at all.

To send Im using
var options = { 
  client_id: settings.push.windows.clientId,
  client_secret: settings.push.windows.clientSecret  
};
wns.sendToastText02(channel, {
        text1 : title,
        text2 : message
      },
      options,
      callback);
    };

on the client it calls the httpnotificationreceived event not the toastnotificationreceived event.

**Comment by matteocontrini** [comment]
In fact it is.

**Comment by elliotstokes** [comment]
I am also using the cordova push plugin. I thought that mpns was going to be retired to assumed I needed wns. Waht mpns module are you using https://github.com/jeffwilcox/mpns? or another.

**Comment by joviol** [comment]
Yes that is the one I am using and it is pretty straightforward to use.
The limitation of working with mpns is that without a TLS certificate, there is a limit of 500 push in a day.
Hope that helps.
